### DISCUSSION TITLE
send captcha in discord group ? no in private message

### DISCUSSION THREAD

**Discussion by polzy** [jef/streetmerchant#2810] [Help]
Hi, 

Is it possible to send the captcha image to a discord group? Rather than in a private message?

thanks you

**Comment by Mr-Tech-13** [comment]
Not that I know of, but you could possibly modify it

**Comment by Mr-Tech-13** [comment]
Yeah, so from what I found, it is relying on a response from a specific user, not just anybody. If you are trying to use it so that anyone can respond to the captcha.

**Comment by polzy** [comment]
I have tried several times, with my abilities I am very limited, I am a powershell developer xD
I was not successful .. :(
